scripts/stratis_id.py

```diff
@@ -122,9 +121,9 @@ def dump_stratis_signature_area(block_device):
 
     dump_stratis_signature_area(sys.argv[1])
 
-    sig = stratis_signature(sys.argv[1])
```

Comment:
For the code in the ```if __name__ == "__main__":``` define a main method and just invoke that, as

```
if __name__ == "__main__":
  main()
```

That way, the code is better and pylint doesn't insist that those identifiers need to be capitalized like they're constants. (They _are_ constants, according to Python, which is why pylint wants them capitalized. But that's not what they're _supposed_ to be, and writing a main method will fix that.)